tensorbay/cli/log.py

```diff
@@ -171,54 +224,83 @@ def _merge(self, latest_commit: Commit) -> bool:
         else:
             return False
         # Merge branches.
-        del self._sorted_commits[-1]
+        del self._sorted_logs[-1]
         del self._keys[-1]
         return True
 
     def _sort(self) -> None:
         """Sort commits paging list by commit date."""
         # Only one branch exists.
-        if len(self._sorted_commits) == 1:
+        if len(self._sorted_logs) == 1:
             return
 
         # Binary insert.
-        commits = self._sorted_commits.pop()
+        commits = self._sorted_logs.pop()
         del self._keys[-1]
         date = commits[0].committer.date
         index = bisect.bisect_left(self._keys, date)
-        self._sorted_commits.insert(index, commits)
+        self._sorted_logs.insert(index, commits)
         self._keys.insert(index, date)
 
-    def _generate_commits(self) -> Iterator[Commit]:
-        """Get the latest commit in commit list.
+    def generate_commits_and_drafts_logs(self) -> Iterator[str]:
```

Comment:
the word `log` in this function means the string to be printed to the terminal about the commit
which is inconsistent with the `log` in `sorted_logs`?